README.md

```diff
@@ -139,7 +146,8 @@ Creators:
   Geoffrey Anneheim aka geogeo - < geoffrey dot anneheim at free dot fr >
   Olivier Armand aka ExtendeD  - < olivier dot calc at gmail dot com >
 
-Contributors and testers: Adriweb, bsl, critor, Excale, Goplat, hoffa, Jim Bauwens, Legimet, Levak, Lionel Debroux, lkj, tangrs... and many others.
+Contributors and testers: Adriweb, bsl, critor, Excale, Goplat, hoffa, Jim Bauwens, Legimet, Levak, Lionel Debroux, lkj, tangrs... and many others.  
+You can see the full list of contributors [here](https://github.com/ndless-nspire/Ndless/graphs/contributors)
```

Comment:
It's not entirely complete as it only has contributions after the move to git. Maybe just make it a `See also` kind of reference?